dces-drc-integration/src/integrationTest/java/uk/gov/justice/laa/crime/dces/integration/service/TestDataService.java

```diff
@@ -0,0 +1,168 @@
+package uk.gov.justice.laa.crime.dces.integration.service;
+
+import static uk.gov.justice.laa.crime.dces.integration.maatapi.model.fdc.FdcContributionsStatus.SENT;
+import static uk.gov.justice.laa.crime.dces.integration.maatapi.model.fdc.FdcContributionsStatus.WAITING_ITEMS;
+
+import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.justice.laa.crime.dces.integration.client.MaatApiClient;
+import uk.gov.justice.laa.crime.dces.integration.maatapi.model.fdc.FdcContributionsStatus;
+import uk.gov.justice.laa.crime.dces.integration.model.external.CreateFdcContributionRequest;
+import uk.gov.justice.laa.crime.dces.integration.model.external.FdcItem;
+import uk.gov.justice.laa.crime.dces.integration.model.external.UpdateFdcContributionRequest;
+import uk.gov.justice.laa.crime.dces.integration.model.external.*;
+import uk.gov.justice.laa.crime.dces.integration.model.local.FdcAccelerationType;
+import uk.gov.justice.laa.crime.dces.integration.model.external.FdcItem.FdcItemBuilder;
+import uk.gov.justice.laa.crime.dces.integration.model.local.FdcItemType;
+import uk.gov.justice.laa.crime.dces.integration.model.local.FdcTestType;
+
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class TestDataService {
+
+    private static final String USER_AUDIT = "DCES";
```

Comment:
These two constants are not used anymore, perhaps they should be, to keep the values in one place?

Reply:
Fixed!